lib/grizzly/inclusions/inclusion_runner/inclusion.ex

```diff
@@ -112,8 +112,9 @@ defmodule Grizzly.Inclusions.InclusionRunner.Inclusion do
   def next_command(inclusion, :dsk_set, seq_number, command_params) do
     dsk = Keyword.fetch!(command_params, :dsk)
     input_dsk_length = byte_size_for_int(dsk)
```

Comment:
Why is the length measured in bytes rather than digits?

Reply:
Because the (partial) DSK is not provided via the Grizzly :node_add_dsk_set as a sequence of digits but as an integer that needs to fit into a required N bytes.

Reply:
That's sounds like a bug in Grizzly. I'll look over the other issues, but it looks like discussing live may be needed.

Reply:
@fhunleth did you want to discuss this more before I merge this?

Reply:
Is it fixed yet? I still see `byte_size_for_int/1` and I thought the plan was to delete that.

Reply:
The update that was just pushed looks good.